config/locales/models/activity.en.yml

```diff
@@ -346,7 +346,7 @@ en:
         objectives: What are the aims/objectives of this %{level}?
         oda_eligibility_lead: Who is the ODA Eligibility Contact for this activity?
         oda_eligibility: Is this activity ODA eligible?
-        commitment: Original commitment figure
+        commitment: Original commitment figure%{optional}
```

Comment:
Ah, here it is - this needs shifting to the previous commit

Reply:
Oops, will move this